framework/core/src/Install/Console/UserDataProvider.php

```diff
@@ -42,20 +42,31 @@ public function configure(Installation $installation): Installation
 
     private function getDatabaseConfiguration(): DatabaseConfig
     {
-        $host = $this->ask('Database host (required):');
-        $port = 3306;
+        $driver = $this->ask('Database driver (mysql, sqlite, pgsql) (Default: mysql):', 'mysql');
+        $port = match ($driver) {
```

Comment:
Do we surface these default ports in the UI?

Reply:
The UI allows passing a port into the host by adding `:3306` for example. Otherwise no, there is no port input with the default port displayed.